cookbooks/adiabatic_boundary.prm

```diff
@@ -0,0 +1,80 @@
+set Dimension                              = 3
+
+set Use years in output instead of seconds = true
+set End time                               = 0
+set Nonlinear solver scheme                = single Advection, single Stokes
+set Nonlinear solver tolerance             = 1e-5
+set Linear solver tolerance                = 1e-5
+set Max nonlinear iterations               = 10 
+set Temperature solver tolerance           = 1e-4
+set Number of cheap Stokes solver steps    = 0
+
+set Pressure normalization                 = surface
+set Surface pressure                       = 0
+
+subsection Geometry model
+  set Model name = ellipsoidal chunk
+  
+  subsection Ellipsoidal chunk
+   set  NE corner = 131:-28
+   set  NW corner = 109:-28
+   set  SW corner = 109:-41
+   set  SE corner =  
+  end
+end   
+
+   
+subsection Initial temperature model
+  set Model name = adiabatic boundary
+  subsection Adiabatic boundary
+    set Isotherm depth filename = adiabatic_boundary.txt
+  end
+end
+
+
+subsection Boundary temperature model
+  set List of model names = box
+
+  subsection Spherical constant
+    set Inner temperature = 4273 # default: 6000
+    set Outer temperature = 273.15 # default: 0
+  end
+end
+
+subsection Model settings
+  set Fixed temperature boundary indicators   =
+  set Zero velocity boundary indicators       =  
+  set Prescribed velocity boundary indicators =
+  set Tangential velocity boundary indicators = 0,1,2,3,4,5 
+
+end
+
```

Comment:
This entire section no longer exists -> please move each parameter to the relevant new section.